### ISSUE TITLE
Browser build

### ISSUE THREAD

**Issue by nullpilot** [kchapelier/poisson-disk-sampling#6]
Using the lib in the browser didn't seem to be an option yet so I put together a quick build to get it working with JSDelivr for now. Maybe that can be leveraged as a starting point.

**Comment by kchapelier** [comment]
As stated in the PR, thanks for your input, having a browser build directly in the repo would indeed be useful. However I'd rather have a browserify-based build process as I'm more confortable with this tool and it is much leaner. I'll tend to it later this week. Sorry for the time you spent on this, I consider it my failing for not providing contribution guidelines...

**Comment by nullpilot** [comment]
No problem at all. I specifically created the issue to get the conversation started. The PR was more of a show and tell to go along with that

**Comment by kchapelier** [comment]
Browser builds have been added to the repo and the readme been updated with instructions on how to use CDNs to load the library. This should significantly ease the use of the library in the browser without requiring the user to setup a build process on his/her end. I made sure the CDN are serving the files correctly with a quick test on Codepen, so everything should be alright but let me know if something is amiss.
